pkg/pgmodel/extension.go

```diff
@@ -23,11 +25,42 @@ var (
 // checkExtensionVersion checks for the correct version and enables the extension if
 // it is at the right version
 func checkExtensionVersion(conn *pgxpool.Pool) error {
-	currentVersion, isInstalled, err := fetchInstalledExtensionVersion(conn)
+	timescaleVersion, isInstalled, err := fetchInstalledExtensionVersion(conn, "timescaledb")
```

Comment:
Nit: would prefer to break up this function into 2: one for each extension

Reply:
That would be good.